### ISSUE TITLE
M3U files should map the disk also to multiple drives

### ISSUE THREAD

**Issue by goedzo** [libretro/libretro-uae#38]
In a lot of amiga games, mapping multiple disks, to multiple drives reduces the amount of disk-swapping required when playing a game. In the UAE file, this was achieved by configuring multiple floppies to multiple drives like this:

floppy0=./././../../Roms/Commodore - Amiga/Dune II - Battle for Arrakis (1993)(Virgin)(M3)[cr PDX][a](Disk 1 of 6).adf
floppy1=./././../../Roms/Commodore - Amiga/Dune II - Battle for Arrakis (1993)(Virgin)(M3)[cr PDX][a](Disk 2 of 6).adf
floppy2=./././../../Roms/Commodore - Amiga/Dune II - Battle for Arrakis (1993)(Virgin)(M3)[cr PDX][a](Disk 3 of 6).adf
floppy3=./././../../Roms/Commodore - Amiga/Dune II - Battle for Arrakis (1993)(Virgin)(M3)[cr PDX][a](Disk 4 of 6).adf
floppy4=./././../../Roms/Commodore - Amiga/Dune II - Battle for Arrakis (1993)(Virgin)(M3)[cr PDX][a](Disk 5 of 6).adf
floppy5=./././../../Roms/Commodore - Amiga/Dune II - Battle for Arrakis (1993)(Virgin)(M3)[cr PDX][a](Disk 6 of 6)(Save).adf

It would be great with the m3u support, also the same mapping is applied so that all the disks in the m3u are also mapped to a new drive.

**Comment by DamsRoxxx** [comment]
From the doc (configuration.txt), P-UAE supports only 4 drive (floppy0 to floppy3):

>Up to four floppy drives may be emulated, numbered 0 to 3,
corresponding to the AmigaDOS devices DF0: to DF3:, respectively. Drive 0 is
the first drive, and drive in which you would typically insert a floppy disk
to boot from.

Does floppy5 and floppy6 worked for you ?

Technically it could be done but disk control interface of RetroArch only support one drive so only the disk in the first drive could be changed later. IMHO, it wouldn't be that much better.

Why not using a WHDLoad version when a game require that much disks ?

**Comment by goedzo** [comment]
Good question and to be honest, I didn't know how to use WHDLoad :). The files I have are all disk files of a game, so not sure how to make a WHDLoad of them. Nevertheless, this discussion extends the purpose of adding more disks to more drives. I don't know if floppy 4 and 5 work, however never got an error in P-UAE. I did have a lot of other issues with the original config load, so I switched to m3u load and this build instead, to solve these issues.
